lib/restaurant.rb

```diff
@@ -0,0 +1,46 @@
+class Restaurant
+  m = Menu.new
+  o = Order.new
```

Comment:
may be wrong, but shouldn't Restaurant initialize with both of these variables? I.e. 
def initialize
  @m = Menu.new
...